src/calendar.cpp

```diff
@@ -536,3 +537,19 @@ bool calendar::once_every(int event_frequency) {
     return (calendar::turn % event_frequency) == 0;
 }
 
+const std::string &calendar::name_season( season_type s )
+{
+    static const std::array<std::string, 5> season_names = {{
+        std::string( _( "Spring" ) ),
+        std::string( _( "Summer" ) ),
+        std::string( _( "Autumn" ) ),
+        std::string( _( "Winter" ) ),
+        std::string( _( "End times" ) )
```

Comment:
Nice.

Reply:
Funny, but impossible to normally get it.